CoreUtilities/src/au/gov/asd/tac/constellation/utilities/log/LogPreferences.java

```diff
@@ -29,6 +31,17 @@ public class LogPreferences {
     private static final Preferences PREFERENCES = NbPreferences.forModule(LogPreferences.class);
     private static final String CONNECTION_LOG_DATE_PREF = "connectionLogDate";
     private static final long LOGGING_TIMEOUT = 151*60*1000L; // 2.5 hours
+    private static final Logger LOGGER = Logger.getLogger(LogPreferences.class.getName());
+    
+    
+    static {
+        if( LOGGER.getUseParentHandlers() ){
+            final Handler[] parentHandlers = LOGGER.getParent().getHandlers();
+            for (final Handler handler : parentHandlers) {
+                handler.setFormatter(new ConstellationLogFormatter());
+            }
+        }
+    }
```

Comment:
This code doesn't get called until the Data Access View is opened.
So, on a clean build with no Views loaded on startup, the ConstellationLogFormatter will not be used until after the Data Access View is opened.
Probably best to move this code to a class that is always accessed on startup.